src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1038,21 +1395,10 @@ MulticopterPositionControl::control_manual(float dt)
 	}
 
 	if (_vehicle_land_detected.landed) {
-		/* don't run controller when landed */
-		_reset_pos_sp = true;
-		_reset_alt_sp = true;
-		_mode_auto = false;
-		_reset_int_z = true;
-		_reset_int_xy = true;
-
-		_R_setpoint.identity();
-
-		_att_sp.roll_body = 0.0f;
-		_att_sp.pitch_body = 0.0f;
-		_att_sp.yaw_body = _yaw;
-		_att_sp.thrust = 0.0f;
-
-		_att_sp.timestamp = hrt_absolute_time();
+		/* don't run controller when landed
+		 * NOTE:
+		 * This only works in manual since once we give throttle input, the
+		 * landdetector will exit the landing state */
```

Comment:
I don't think it does. Only XY position control get disabled because they produced the biggest problem with tip overs and too high thrust when trying to land because the vehicle does not move anymore.